messaging/ratelimits.md

```diff
@@ -4,15 +4,6 @@
 
 This guide will walk through the recommended approach to managing queues and rate limits for use with the Messaging API. Over the past years mobile telecom operators have begun to block what is deemed automated traffic (**A2P**) sent over standard local telephone numbers IE: (919)-430-5555. The amount of messages sent in this way have increased due to spreading automated traffic (A2P) across multiple local telephone numbers to bypass volumetric filters. This process is called "snowshoeing" and as a result, the mobile operators are not only blocking volumetrically, but are also finger-printing content and preemptively blocking messages even from a "fresh" phone number.
 
-## Assumptions
```

Comment:
Let's leave the assumptions about CTIA & TCPA. those are frequently sent in sales as they govern how we are able to operate.

Reply:
Sounds good. Changed